app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import app.errors
```

Comment:
Import the exact error classes from `app.errors` that you will need to call, it is a good practice to import classes instead of calling them from the module in this case. This comment relates to `app/main.py` as well.